typed_sql/lib/src/dialect/sqlite_dialect.dart

```diff
@@ -0,0 +1,624 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import 'dart:typed_data' show Uint8List;
+
+import 'package:collection/collection.dart';
+
+import 'dialect.dart';
+
+SqlDialect sqliteDialect() => _Sqlite();
```

Comment:
dartdoc? Perhaps mention that this covers sqlite3?

Reply:
Done in `lib/dialect.dart` on `SqlDialect.sqlite` where it's exposed.